README.md

```
@@ -58,19 +58,17 @@ EHR2MEDS is a tool that formats dumps of Electronic Health Records (EHR) and con
-
-   Example:
-
-   ```bash
-   python ehr2meds/normalize_premeds.py --config-name preMEDS/normalise     
-   ```
-
-   Example configuration files can be found in the [configs/preMEDS](./ehr2meds/configs/preMEDS).
+### Custom MEDS stages
+
+The package includes the following stages to be used in MEDS pipeline configurations: 
+| Stage | Purpose |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)